magnum_capi_helm/driver.py

```diff
@@ -62,6 +62,11 @@ def provides(self):
                 "os": "capi-kubeadm-cloudinit",
                 "coe": "kubernetes",
             },
+            {
+                "server_type": "vm",
+                "os": "flatcar",
+                "coe": "kubernetes",
+            },
```

Comment:
will this clash with https://github.com/vexxhost/magnum-cluster-api/blob/9d10552125494bcf3bffafea35429f6c6439b1f0/magnum_cluster_api/driver.py#L395 ?

Reply:
Yes, but that needs to be solved by the Magnum project, not awkwardly avoided by the CAPI drivers.